### PR TITLE
builddep: properly check for nosrc.rpm

### PR DESCRIPTION
**Pull request by ignatenkobrain**
Signed-off-by: Igor Gnatenko <ignatenko@redhat.com>

### CODE REVIEW

File: plugins/builddep.py [hunk 1]
```diff
@@ -94,9 +94,9 @@ def configure(self):
         # enable source repos only if needed
         if not (self.opts.spec or self.opts.srpm):
             for pkgspec in self.opts.packages:
-                if not (pkgspec.endswith('.src.rpm')
-                        or pkgspec.endswith('nosrc.rpm')
-                        or pkgspec.endswith('.spec')):
+                if not (pkgspec.endswith('.src.rpm') or
```
Comment (j-mracek) [inline plugins/builddep.py]: @ignatenkobrain I am not sure about corret place of operator. Acording to (https://github.com/PyCQA/pycodestyle/issues/197), operator should be at beginning of next line. Please can you provide an evidence? The rest is fine.

Comment (ignatenkobrain) [inline plugins/builddep.py]: reverted

Comment (j-mracek) [inline plugins/builddep.py]: What was reverted?

Comment (ignatenkobrain) [inline plugins/builddep.py]: I reverted change of breaking around binary operators. And added it to flake ignore

**Comment by j-mracek** [comment]
@dnf-bot r+

**Pull request closed without merging**